app/views/saasbook/_topbar.html.erb

```diff
@@ -0,0 +1,88 @@
+<div class="topbar container-xl fixed-top">
+    <div class="topbar-contents row">
+        <div class="col-12 col-md-3 bd-topbar-whitespace site-navigation show"></div>
+        <div class="col pl-md-4 topbar-main">
+            
+            <button id="navbar-toggler" class="navbar-toggler ml-0" type="button" data-toggle="collapse"
+                data-toggle="tooltip" data-placement="bottom" data-target=".site-navigation" aria-controls="navbar-menu"
+                aria-expanded="true" aria-label="Toggle navigation" aria-controls="site-navigation"
+                title="Toggle navigation" data-toggle="tooltip" data-placement="left">
+                <i class="fas fa-bars"></i>
+                <i class="fas fa-arrow-left"></i>
+                <i class="fas fa-arrow-up"></i>
+            </button>
+            
+            
+            <div class="dropdown-buttons-trigger">
+                <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn" aria-label="Download this page"><i
+                        class="fas fa-download"></i></button>
+
+                <div class="dropdown-buttons">
+                    <!-- ipynb file if we had a myst markdown file -->
+                    
+                    <!-- Download raw file -->
+                    <a class="dropdown-button" href="#" download>
+                        <button type="button"
+                            class="btn btn-secondary topbarbtn" title="Download source file" data-toggle="tooltip"
+                            data-placement="left">.html
+                        </button>
+                    </a>
+                    <!-- Download PDF via print -->
+                    <a class="dropdown-button">
+                        <button type="button" id="download-print" class="btn btn-secondary topbarbtn" title="Print to PDF"
+                            onClick="window.print()" data-toggle="tooltip" data-placement="left">.pdf
+                        </button>
+                    </a>
+                </div>
+            </div>
+
+            <!-- Source interaction buttons -->
+            <a class="annotation-button" ><button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
+                    data-placement="bottom" title="Make annotation"><i

```

Comment:
I think for these we should just label as highlight and annotate, keeps wording consistent.

Reply:
Yeah, I think it's better that way. Maybe can add comments for functionality for newly added buttons like the ones for downloads? It'll make code more readable and easier to understand